JavaScriptSPA/index.html

```diff
@@ -65,46 +89,52 @@ <h4 id="WelcomeMessage"></h4>
         xmlHttp.send();
     }
 
+
     function graphAPICallback(data) {
-        //Display user data on DOM
-        var divWelcome = document.getElementById('WelcomeMessage');
-        divWelcome.innerHTML += " to Microsoft Graph API!!";
-        document.getElementById("json").innerHTML = JSON.stringify(data, null, 2); 
+        document.getElementById("json").innerHTML = JSON.stringify(data, null, 2);
```

Comment:
Is this the user data that we see once login()? Can this be spruced up in the UI? I understand we are not writing an application but may be making it less blander is a good idea later?? :-)

Reply:
I thought about it, but I actually think for developers, seeing json is valuable